cmd/root.go

```diff
@@ -48,20 +48,19 @@ var rootCmd = &cobra.Command{
 		env.InitLogger()
 
 		source := env.Config().GetString("Source")
-		outputDir := env.Config().GetString("OutputDir")
 
 		configs := []ocp.Translator{}
-		ocpMaster := new(ocp.ConfigMaster)
-		ocpNode := new(ocp.ConfigNode)
-		ocpMaster.Add(source)
-		ocpNode.Add(source)
-		configs = append(configs, ocpMaster, ocpNode)
+		ocpOAuth := new(ocp.OAuthConfig)
+		ocpSDN := new(ocp.SDNConfig)
+		ocpOAuth.Add(source)
```

Comment:
This is absolutely right, because of various undecided sources of endpoint including input for user (now deprecated, so far). 
This needs to be worked it.